CHANGELOG.md

```diff
@@ -1,138 +1,117 @@
 
 Bug fixes:
 
-- Trap exceptions from socket.send() (#137) - thanks @frankgreco
-- Use ip tag instead of peer.ipv4 in process (#125)
-- Log only span context (#153)
-- Fix buffer size counting bug and add logging to UDP sender (#151)
-
+* Trap exceptions from socket.send() (#137) - thanks @frankgreco
+* Use ip tag instead of peer.ipv4 in process (#125)
+* Log only span context (#153)
+* Fix buffer size counting bug and add logging to UDP sender (#151)
 
-3.5.3 (2017-04-24)
-------------------
+## 3.5.3 (2017-04-24)
 
-- Fix SamplingStrategyResponse strategyType (#110)
+* Fix SamplingStrategyResponse strategyType (#110)
 
+## 3.5.2 (2017-03-29)
 
-3.5.2 (2017-03-29)
-------------------
+* Protect from exceptions in decodeURIComponent (#105)
+* Do not url-encode span context (#105)
 
-- Protect from exceptions in decodeURIComponent (#105)
-- Do not url-encode span context (#105)
+## 3.5.1 (2017-03-07)
 
+* Fix bug where leaky bucket never fills up when creditsPerSecond < itemCost (#104)
 
-3.5.1 (2017-03-07)
-------------------
+## 3.5.0 (2017-03-06)
 
-- Fix bug where leaky bucket never fills up when creditsPerSecond < itemCost (#104)
+* Remove dependency on 'request' module, use 'http' instead (https://github.com/uber/jaeger-client-node/pull/103)
 
+## 3.4.0 (2017-02-20)
 
-3.5.0 (2017-03-06)
-------------------
+* Allow tags/logs with 'object' values and convert them to JSON (#102)
 
-- Remove dependency on 'request' module, use 'http' instead (https://github.com/uber/jaeger-client-node/pull/103)
+## 3.3.1 (2017-02-13)
 
+* Add TestUtils.getTags(span, ?keys)
 
-3.4.0 (2017-02-20)
-------------------
+## 3.3.0 (2017-02-09)
 
-- Allow tags/logs with 'object' values and convert them to JSON (#102)
+* Make Configuration accept MetricsFactory, not Metrics
 
+## 3.2.1 (2017-02-08)
 
-3.3.1 (2017-02-13)
-------------------
+* Make sure initTracer passes options to the tracer
+* Do not wrap single RemoteReporter into CompositeReporter
 
-- Add TestUtils.getTags(span, ?keys)
+## 3.2.0 (2017-02-04)
 
+* Remove the following dependencies
+  * `"deep-equal": "^1.0.1",`
+  * `"long": "^3.2.0",`
+  * `"js-yaml": "^3.6.1",`
+  * `"jsonschema": "^1.1.0",`
+* Move `TestUtil.thriftSpansEqual` and `Util.combinations` functions under `tests/lib`
+* Remove most methods from TestUtils because the same checks can be done via public API
+* Remove `hasLogs` method that was not particularly useful in practice because it compared the timestamp
+* Accept external timestamps in milliseconds since epoch (#94)
+* Expose TChannelBridge.inject method (#93)
 
-3.3.0 (2017-02-09)
-------------------
+## 3.1.0 (2017-02-02)
 
-- Make Configuration accept MetricsFactory, not Metrics
+* Added support for end to end crossdock integration tests.
+* Fixed bug where "peer.ipv4" tag was not being saved.
+* Fixed bug where tracer tags were being reported twice.
+* Updated sampler config to allow extra customization.
 
+  3.0.0 (2017-01-20)
```

Comment:
missing `##` here and a few versions below

Reply:
Oups..